cli/clibase/cmd.go

```diff
@@ -55,6 +60,14 @@ func (c *Cmd) Walk(fn func(*Cmd)) {
 	}
 }
 
+// SetParents sets the Parent field of all children.
+func (c *Cmd) SetParents() {
```

Comment:
nit: `ConfigureChildren` makes more sense to me. This name makes it seem like we're setting parents of the root struct, which isn't true.

Reply:
`ConfigureChildren` implies a broader scope than just setting a parent field.